.github/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+# Description
+
+*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.*
+
+## Related Issue(s)
+
+- Issue: #<issue_number_here>
+- See also:
+    - #<issue_number>
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation
+- [ ] Other (please describe):
+
+# Changes
+
```

Comment:
I assume that the format here follows the same style/format as the changelog. In that case, I think we should add an instruction to indicate that, maybe with the same link to `Keep a changelog`.